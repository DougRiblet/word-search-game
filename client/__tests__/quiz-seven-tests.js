import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';
import CurrentSeven from '../components/Quiz-Seven/quiz-seven-present';

describe('Quiz Seven', () => {
  test('Snapshot', () => {
    const CurrentSeven_tree = renderer.create(
      <CurrentSeven currentseven='JEEZELY' />
    ).toJSON();
    expect(CurrentSeven_tree).toMatchSnapshot();
  });

  test('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<CurrentSeven currentseven='JEEZELY' />, div);
  });

  test('displays current seven-letter word', () => {
    const wrapper = shallow(<CurrentSeven currentseven='JEEZELY' />);
    expect(wrapper.text()).toEqual('JEEZELY');
    expect(wrapper.html()).toEqual('<div id="quiz-seven"><h1>JEEZELY</h1></div>');
  });

});
