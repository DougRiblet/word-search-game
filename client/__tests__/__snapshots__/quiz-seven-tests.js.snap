// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quiz Seven Snapshot 1`] = `
<div
  id="quiz-seven"
>
  <h1>
    JEEZELY
  </h1>
</div>
`;
